region/tourian/main/Tourian Escape Room 3.json

```diff
@@ -331,6 +335,42 @@
         "A speedy jump can reach the top platform."
       ]
     },
+    {
+      "link": [1, 2],
+      "name": "Use Flash Suit",
+      "requires": [
+        {"obstaclesCleared": ["A"]},
+        {"useFlashSuit": {}},
+        {"shinespark": {"frames": 7, "excessFrames": 5}}
+      ],
+      "flashSuitChecked": true,
+      "note": [
+        "Safely kill the first pirate, then walk through the rest while taking damage or by using Plasma Beam.",
+        "It is possible to use the camera to manipulate the pirates from below so they are closer together, then run through them to save a hit."
+      ]
+    },
+    {
+      "link": [1, 2],
+      "name": "Use Flash Suit, Tank the Damage",
+      "requires": [
+        "canCameraManip",
+        {"enemyDamage": {
+          "enemy": "Tourian Space Pirate (all)",
+          "type": "contact",
+          "hits": 1
+        }},
+        {"useFlashSuit": {}},
+        {"shinespark": {"frames": 7, "excessFrames": 5}},
+        {"enemyDamage": {
+          "enemy": "Tourian Space Pirate (all)",
+          "type": "contact",
+          "hits": 1
+        }}
+      ],
+      "flashSuitChecked": true,
+      "note": "Use the camera to manipulate the pirates from below so they are closer together, then run through them to save a hit.",
```

Comment:
Shouldn't this be going through the speed blocks?

Reply:
Suitless it saves 3 energy to go through the speed blocks. With a suit this way saves 5 energy, with both 9 energy. 

I'm thinking I'll remove this one and switch it to through the speed blocks. You can kill the pirates freely with patience, so if you don't have patience turned on, I dont see an elevator CF to get a speedless flash suit as a good alternative.